core/src/test/java/io/aiven/klaw/UsersTeamsControllerIT.java

```diff
@@ -633,7 +636,7 @@ public void showUserList() throws Exception {
             .getContentAsString();
     List<UserInfoModelResponse> userInfoModelList =
         new ObjectMapper().readValue(response, new TypeReference<>() {});
-    assertThat(userInfoModelList).hasSizeBetween(3, 6); // superadmin, kwuserb, kwuserc
+    assertThat(userInfoModelList).hasSizeBetween(3, 9); // superadmin, kwuserb, kwuserc
```

Comment:
Should this be a bit more precise, I am assuming it should equal the same amount every time, or use assertThat(userInfoModelList).containsAll(a,b,c,d) ?

Reply:
We added this as this test might run before or after other integration tests, and we are not cleaning up users after the integration test. 
So there are about 9 users created totally.

Reply:
Ok I think we can maybe improve this later, it doesn't need to hold up this PR